### ISSUE TITLE
Nick and AltNick don't work, when different case of global nickname

### ISSUE THREAD

**Issue by marumari** [znc/znc#1182]
In my global setting, I have my IRC nickname set to "April".  However, on one network, I'd like it to be "april" (lowercase).

If I try to set Nick and AltNick to "april" in the network-specific configuration page, it just resets back to the default of "April".  If I add them directly to znc.conf, they get ignored and deleted upon startup.

**Comment by DarthGandalf** [comment]
I think this is correct behavior.
AltNick is used when ZNC can't use the primary nick when connecting to server (e.g. if the nick is used by someone else). If "April" can't be used, "april" can't be used either, because IRC protocol compares nicks case-insensitively.

**Comment by marumari** [comment]
Right, but April and april both can be used; that's not the problem.  I want my nick to be lowercase on just one network, but ZNC ignores setting it at a network level if it's the same name (but a different case) of the global nick.

**Comment by PlasmaStar** [comment]
I've experienced the same situation. There's some network(s) I'd like to have a different casing nick on too. As a work around, I have my perform module /nick to the different casing, but I'd much rather see the network specific configuration handle the different casing request. For example, I'm using PlasmaStar on most networks, but on at least one network, I'd like it to be plasmaStar, without having to use some auto-perform line.

**Comment by wowaname** [comment]
time to upgrade to weechat
